CONTRIBUTING.md

```diff
@@ -0,0 +1,242 @@
+# How to contribute to NumaHOP.
+
+## Did you find a bug ?
+
+- Do not open an issue if the bug is a security vulnerability in our dependencies.
+- Ensure the bug was not already reported by searching the [issues](https://github.com/biblibre/NumaHOP-code/issues).
+- If your unable to find an open issue addressing the problem, [open a new one](https://github.com/biblibre/NumaHOP-code/issues/new).
+Be sure to include a **title and clear description** with as much relevant information as possible.
```

Comment:
What is a clear description? What information is relevant? Maybe some examples would be useful here.

Reply:
See https://wiki.koha-community.org/wiki/Bug_Reporting_Guidelines